akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -200,6 +200,12 @@ To disable passivation you can use configuration:
 akka.cluster.sharding.passivate-idle-entity-after = off
 ```
 
+### waiting-for-state-timeout reduced to 2s
+
+This has been reduced to speed up ShardCoordinator initialization in smaller clusters.
+The read from ddata is a ReadMajority, for small clusters (< majority-min-cap) every node needs to respond
+so is more likely to timeout if there are nodes restarting i.e. when there is a rolling re-deploy happening.
```

Comment:
Maybe we should have a separate section in migration guide about configuration (and behavior) changes that doesn't require any "migration" changes from the user?

On the other hand it's also nice to group things per module (Sharding, Streams).

Reply:
Useful to add to the migration guide and note in the respective doc sections.

Reply:
I think a separate section for configuration changes makes sense

Reply:
I restructured the migration guide to put all changes of behavior that don't require user action into their own section so we have fewer top-level headings